docs/core/tools/dotnet.md

```diff
@@ -5,212 +5,157 @@ ms.date: 02/13/2020
 ---
 # dotnet command
 
-[!INCLUDE [topic-appliesto-net-core-all](../../../includes/topic-appliesto-net-core-all.md)]
+**This article applies to:** ✔️ .NET Core 2.1 SDK and later versions
 
 ## Name
 
-`dotnet` - A tool for managing .NET source code and binaries.
+`dotnet` - The generic driver for the .NET Core CLI.
 
 ## Synopsis
 
-<!-- markdownlint-disable MD025 -->
-
-# [.NET Core 2.1](#tab/netcore21)
+To get information about the available commands and the environment:
 
 ```dotnetcli
-dotnet [command] [arguments] [--additional-deps] [--additionalprobingpath] [--depsfile]
-    [-d|--diagnostics] [--fx-version] [-h|--help] [--info] [--list-runtimes] [--list-sdks] [--roll-forward-on-no-candidate-fx] [--runtimeconfig] [-v|--verbosity] [--version]
+dotnet [-h|--help] [--version] [--info]
+    [--list-runtimes] [--list-sdks]
 ```
 
-# [.NET Core 2.0](#tab/netcore20)
+To run a command:
```

Comment:
This one basically requires SDK - the only command which works without SDK is `dotnet exec` which is special - and probably can be added to the case below (as it runs an app).